ui/app/pages/send/send-content/send-gas-row/send-gas-row.component.js

```diff
@@ -156,16 +154,19 @@ export default class SendGasRow extends Component {
   }
 
   render () {
-    const { gasFeeError } = this.props
+    const { gasFeeError, gasButtonGroupShown } = this.props
 
     return (
-      <SendRowWrapper
-        label={`${this.context.t('transactionFee')}:`}
-        showError={gasFeeError}
-        errorType="gasFee"
-      >
-        { this.renderContent() }
-      </SendRowWrapper>
+      <>
+        <SendRowWrapper
+          label={`${this.context.t('transactionFee')}:`}
+          showError={gasFeeError}
+          errorType="gasFee"
+        >
+          { this.renderContent() }
+        </SendRowWrapper>
+        {gasButtonGroupShown && <SendRowWrapper>{ this.renderAdvancedOptionsButton() }</SendRowWrapper>}
```

Comment:
Do you recall why these changes were made? I don't remember this part.

Reply:
Phew, found [it](https://github.com/MetaMask/metamask-extension/pull/9074#discussion_r465990095) - slight improvement of how items are grouped to allow for better alignment

Reply:
Ah, thanks! Yes I see why you did this now - this is much better.

This does seem to be a change in logic though. Previously these buttons were also shown when `advancedInlineGasShown` was truthy. e.g.

```suggestion
        {
          gasButtonGroupShown || advancedInlineGasShown
            ? <SendRowWrapper>{ this.renderAdvancedOptionsButton() }</SendRowWrapper>
            : null
        }
```

(though this suggestion can't be accepted directly - the `advancedInlineGasShown` prop needs to be brought into scope.

Reply:
ah thanks!

Reply:
updated in last commit